### ISSUE TITLE
[8.0.0-beta-3 - Nuxt 3 RC12] Vue router throw warnings to a console when I use strategy: "prefix"

### ISSUE THREAD

**Issue by cryptopay-pavel** [nuxt-modules/i18n#1602]
### Version
@nuxtjs/i18n: 8.0.0-beta.3-27787267.945868d
nuxt: 3.0.0-rc.12

### Nuxt configuration

Please change to `[x]` if relevant for this issue:

 - [ ] Applies to a site deployed to a static server (site generated with `nuxt generate`)
 - [x] Applies to a site deployed to a server with a Node backend

### @nuxtjs/i18n configuration

```
i18n: {
    debug: false,
    strategy: "prefix",
    langDir: "locales",
    locales: [
      {
        code: "en", iso: "en-GB", file: "en_GB.json", dir: "ltr"
      },
      {
        code: "eu", iso: "EEA", file: "en.json", dir: "ltr"
      },
      {
        code: "es", iso: "ES", file: "es.json", dir: "ltr"
      },
      {
        code: "fr", iso: "fr-FR", file: "fr.json", dir: "ltr" 
      },
      {
        code: "de", iso: "DE", file: "de.json", dir: "ltr" 
      },
    ],
    defaultLocale: "en",
    vueI18n: {
      locale: "en",
      fallbackLocale: "en",
      legacy: false,
    }
  },
```

### What is Expected?
@nuxtjs/i18n module successfully creates localised routes without warnings in terminal.

### What is actually happening?
@nuxtjs/i18n module successfully creates localised routes but with many warnings in terminal.

It happens only with 'strategy: "prefix"' parameter. If I use "prefix_and_default" and router doesn't create routes for default locale - warnings disappear.

### Log:

```
ℹ Using Tailwind CSS from ~/assets/css/tailwind.css                      nuxt:tailwindcss 21:59:04
ℹ Tailwind Viewer: http://[::]:3000/_tailwind/                           nuxt:tailwindcss 21:59:04
ℹ Vite client warmed up in 7829ms                                                         21:59:13
✔ Nitro built in 660 ms                                                             nitro 21:59:13
Nuxi 3.0.0-rc.12                                                                          21:59:13
                                                                                          21:59:13
  > Local:    http://localhost:3000/
  > Network:  http://192.168.100.5:3000/

[Vue Router warn]: No match found for location with path "/"
[Vue Router warn]: No match found for location with path "/"
[Vue Router warn]: No match found for location with path "/"
[Vue Router warn]: No match found for location with path "/"
[Vue Router warn]: No match found for location with path "/"
[Vue Router warn]: No match found for location with path "/"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/"
[Vue Router warn]: No match found for location with path "/"
[Vue Router warn]: No match found for location with path "/"
[Vue Router warn]: No match found for location with path "/"
[Vue Router warn]: No match found for location with path "/"
[Vue Router warn]: No match found for location with path "/"
[nitro] [dev] [unhandledRejection] Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:387:5)
    at ServerResponse.setHeader (node:_http_outgoing:603:11)
    at Object.errorhandler [as onError] (file:///Users/pas/cryptopay/redesign-landings/.nuxt/dev/index.mjs:440:13)
    at async Server.toNodeHandle (file:///Users/pas/cryptopay/redesign-landings/node_modules/h3/dist/index.mjs:638:9) {
  code: 'ERR_HTTP_HEADERS_SENT'
}
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
[Vue Router warn]: No match found for location with path "/crypto-wallet"
[Vue Router warn]: No match found for location with path "/buy-cryptocurrency"
[Vue Router warn]: No match found for location with path "/crypto-card"
[Vue Router warn]: No match found for location with path "/earn-crypto"
[Vue Router warn]: No match found for location with path "/accounts"
[Vue Router warn]: No match found for location with path "/legal"
[Vue Router warn]: No match found for location with path "/referral-programme"
[Vue Router warn]: No match found for location with path "/prices"
```

**Comment by ineshbose** [comment]
Same issue here! (Making use of `useLocalePath()` with links to `pages` along with some `content` files that are fetched using `queryContent`)

> `[nitro] [dev] [unhandledRejection] Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client`

Is this error also due to this module? I'm having this as well, but unable to trace back where it's coming from. (Edit: now #1611)

**Comment by BobbieGoede** [comment]
Resolved by #1612
